component/models/ld/mock/mock_context_store.go

```diff
@@ -13,14 +13,14 @@ import (
 
 	jsonld "github.com/piprate/json-gold/ld"
 
-	"github.com/hyperledger/aries-framework-go/pkg/doc/ldcontext"
-	mockstorage "github.com/hyperledger/aries-framework-go/pkg/mock/storage"
-	"github.com/hyperledger/aries-framework-go/pkg/store/ld"
+	"github.com/hyperledger/aries-framework-go/component/models/ld/context"
+	store "github.com/hyperledger/aries-framework-go/component/models/ld/store"
+	mockstorage "github.com/hyperledger/aries-framework-go/component/storageutil/mock/storage"
 	"github.com/hyperledger/aries-framework-go/spi/storage"
 )
 
 // MockContextStore is a mock JSON-LD context store.
-type MockContextStore struct {
+type MockContextStore struct { // nolint:golint
```

Comment:
Reason for the `nolint`?

Reply:
This is for the name stuttering, but I've now fixed the stuttering instead.